Assets/Game/Scripts/Character/AnimationTrigger.cs

```diff
@@ -1,8 +1,8 @@
-﻿public enum AnimationTrigger
+﻿using UnityEngine;
+
+public class AnimationTrigger
 {
-  Move,
-  Idle,
-  Attack,
-  Die,
-  Hurt
+  public static readonly int dead = Animator.StringToHash("Dead");
+  public static readonly int attack = Animator.StringToHash("Attack");
+  public static readonly int run = Animator.StringToHash("Run");
 }
```

Comment:
Hurt is actually a better name than hit, may be use it next push. Need to change events in animator to match.

Reply:
I just updated the diagram to use hurt instead of hit. We can add hurt to the class when we reach that part.